frontend/app/views/spree/orders/_line_item_data.html.erb

```diff
@@ -22,7 +22,7 @@
 <% end %>
 
 <div class="shopping-cart-item-delete <%= additional_classes %>" data-hook="cart_item_delete">
-  <%= link_to '#', class: 'delete', id: "delete_#{dom_id(line_item)}", data: { turbolinks: false } do %>
+  <%= link_to '#', class: 'delete', id: "delete_#{dom_id(line_item)}", data: { turbolinks: false }, aria: { label: 'Remove from cart' } do %>
```

Comment:
@kaliszKrzysztof can we use translation for this, please?

Reply:
Sure